packages/diagrams/lib/src/theme_data.dart

```diff
@@ -0,0 +1,91 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:diagram_capture/diagram_capture.dart';
+import 'package:flutter/material.dart';
+
+import 'diagram_step.dart';
+
+const String _themeData = 'theme_data';
+const String _materialAppThemeData = 'material_app_theme_data';
+
+class ThemeDataDiagram extends StatelessWidget implements DiagramMetadata {
+  const ThemeDataDiagram(this.name);
+
+  @override
+  final String name;
+
+  @override
+  Widget build(BuildContext context) {
+    Widget returnWidget;
+
+    switch (name) {
+      case _themeData:
+        returnWidget = Theme(
+          data: ThemeData(primaryColor: Colors.amber),
```

Comment:
This shows how to access different properties of the given context's theme. 👍 
Let's do one swatch instead of two. And how about we illustrate the theme inheritance across different widgets with the other sample. You could set things like `primaryColor` and `accentColor`, `buttonColor` etc, and then create a page that uses widgets that are inherit these properties from the theme. WDYT?

Reply:
I dig it. I'll update the example to show that!

Reply:
All right -- I've updated one of the diagrams with a few different `ThemeData` properties and made use of labels to help identify which properties effect specific widgets!

Happy to add more / change it up if you'd like to see something specific.

Reply:
Looks fantastic!